pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>com.esri.geoevent.parent</groupId>
 	<artifactId>reverse-geocoder</artifactId>
-	<version>10.5.0</version>
+	<version>10.6.0</version>
```

Comment:
let's keep upgrading the version to 10.6.0 separate from this PR so we can include this enhancement with 10.5.x